cluster/addons/dns/coredns_README.md

```diff
@@ -0,0 +1,30 @@
+## coredns
+
+Users can enable CoreDNS instead of kube-dns for service discovery.
+`coredns` is schedules DNS Pods and Service on the cluster, other pods in cluster
+can use the DNS Service’s IP to resolve DNS names.
+
+* [Administrators guide](https://coredns.io/2018/01/29/deploying-kubernetes-with-coredns-using-kubeadm/)
```

Comment:
This guide applies to installing CoreDNS via kubeadm and doesn't apply here.

Reply:
@rajansandeep i agree.
also given the ongoing transition to CoreDNS the above paragraph would be misleading for the next release. i would Close this PR, until we have settled.